traffic_monitor/health/cache.go

```diff
@@ -322,7 +327,7 @@ func CalcAvailability(
 			Ipv6Available: availStatus.Available.IPv6,
 		})
 
-		if available, ok := localStates.GetCache(tc.CacheName(result.ID)); !ok || !available.IsAvailable || !availStatus.ProcessedAvailable {
```

Comment:
Originally this was `available.IsAvailable != availStatus.ProcessedAvailable` (4.1). I think it should use that instead of `lastStatus`.
I'm not seeing a log entry when going from ADMIN_DOWN -> ONLINE and I assume this is the reason.

Reply:
`available` no longer exists

Reply:
I am now seeing it from that path - which is to say you are correct